www/js/overlay.js

```diff
@@ -226,17 +233,69 @@ const toggleCast = h(
   h('div.setting-label', `${overlayCast ? 'On' : 'Off'}`),
 );
 
+const toggleLogo = h(
+  'div.input-wrap.logo-toggle',
+  {
+    onclick: evt => {
```

Comment:
analytics events would be useful to measure how many people use logos

Reply:
+1